### PR TITLE
[SA-2262] Add an example of smallest tenant definition to the MTO docs

### PR DESCRIPTION
**Pull request by MahnoorAsghar**
Add an example of the smallest valid Tenant CR definition to the MTO docs

### CODE REVIEW

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-7891890a`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-4a3c4a1f`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-fd336a11`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-7cd581df`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-7d0497e1`

File: docs/content/sre/multi-tenant-operator/customresources.md [hunk 1]
```diff
@@ -66,69 +66,82 @@ For more details [Quota.Spec](https://kubernetes.io/docs/concepts/policy/resourc
...
-      template: networkpolicy
-    parameters:
-      - name: CIDR_IP
-        value: "172.17.0.0/16"
-    selector:
-      matchLabels:
+  templateInstances: # optional
+  - spec: # optional
+      template: networkpolicy # required
+      parameters: # optional
```
Comment (MuneebAijaz) [inline docs/content/sre/multi-tenant-operator/customresources.md:140]: wrong indentation

Comment (MahnoorAsghar) [reply docs/content/sre/multi-tenant-operator/customresources.md:140]: No, I think this is the correct indentation: https://github.com/stakater-ab/tenant-operator/blob/2a52ec7752ed5a82ff4ebaba830d7819cfecde66/api/v1beta2/tenant_types.go#L20

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-08297bf2`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-86-50d4612a`

**Review by MuneebAijaz — APPROVED** [review]

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.67`

**Pull request merged**